app/containers/DocumentList/DocumentList.jsx

```diff
@@ -67,6 +73,11 @@ class DocumentList extends React.Component {
      */
     page: proptypes.number,
 
+    /**
+     * Hashed location key from React Router.
+     */
+    routeKey: proptypes.string,
```

Comment:
Same question here. The reason we introduced `contentKey` was to allow the containers (i.e. `Document` and `DocumentList`) to operate independently of the view. You could have multiple on the same page. So each one gets a `contentKey`, so that they don't need to be aware of the concept of routes. Any reason to introduce this here?

Reply:
Fixed